legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/sqlQueryToString/dbExtension.pure

```diff
@@ -722,7 +724,8 @@ Class meta::relational::functions::sqlQueryToString::DynaFunctionToSql
 
    toSql(parameters:RelationalOperationElement[*], sgcs:SqlGenerationContext[*])
    {
-      let processedParams = $parameters->zip($sgcs)->map(p | $p.first->processOperation($p.second));
+      let processedParams = $parameters->zip($sgcs)->map(p |assert($p.second.config.isBooleanNotAllowed->isEmpty() || $p.second.config.isBooleanNotAllowed == false, '[unsupported-api] Boolean expression is not allowed');
```

Comment:
`assert($p.second.config.isBooleanNotAllowed != true || !$p.first->isBooleanOperation($p.second.extensions), '[unsupported-api] boolean expression is not allowed in non-boolean contexts of dyna fn "' + $this.funcName + '"');`